### PR TITLE
Feature/add option to set dropdown max height

### PR DESCRIPTION
**Pull request by raphaellueckl**
Adds possibility to set a variable height for the dropdown.

### CODE REVIEW

File: src/components/20-molecules/dropdown/story.js [hunk 1]
```diff
@@ -35,6 +36,7 @@ storyDropdown.add('Dropdown', () => {
       name="${name}"
       dataTestId="${dataTestId}"
       error="${error}"
+      maxHeight="${maxHeight}"
```
Comment (01231) [inline src/components/20-molecules/dropdown/story.js:39]: please add `maxHeight` also to the react example

**Review by 01231 — CHANGES_REQUESTED** [review]

File: src/components/20-molecules/dropdown/story.js (hunk 1 above)
Comment (raphaellueckl) [reply src/components/20-molecules/dropdown/story.js:39]: thx

File: src/components/20-molecules/dropdown/react/story.jsx [hunk 2]
```diff
@@ -36,6 +36,7 @@ storiesOf('Molecules/Dropdown/React', module)
     const required = boolean('required', false);
     const checkMark = boolean('checkmark', false);
     const disabled = boolean('disabled', false);
+    const maxHeight = text('max-height', '');
```
Comment (01231) [inline src/components/20-molecules/dropdown/react/story.jsx:33]: `number` and `select` are unused (row 5 & 6 story.jsx)

**Review by 01231 — APPROVED** [review]
Nice!

**Pull request merged**